src/utils/emails.js

```diff
@@ -64,10 +91,11 @@ const sendEmail = async (email, name, heading, res, messageTemplate, token) => {
     await sgMail.send(message);
     // RETURN SUCCESS MESSAGE
     /* eslint-disable no-console */
-    console.log(`Email sent to ${name}`);
+    console.log(message);
```

Comment:
Logging a complete would be lengthy. I would suggest to log only a confirmation that the email has been sent. Similar to what was removed on [line 67](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn/pull/42/files#diff-8258c2a30dd09b705828fbe37cc2599233a09900f067b13da8beb82b80cc2fa6L67).